test/integration/emulator/TimeTurner.cpp

```diff
@@ -100,9 +100,22 @@ void TimeTurner::runFor(uint64_t durationNs)
          timestamp = getAbsoluteTime())
     {
         logger::awaitLogDrained(0.75);
-        if (!_sleeperCountdown.wait(1000))
+        // wait up to 1s for threads to go into nanoSleep
```

Comment:
Sometimes test was stalled temporarily 1s here because a thread was doing ~JobQueue waiting for semaphore.
Now we wait 5ms and release some worker threads by advancing 64ns and thereby let the worker threads allow the waiting thread to continue.